src/main/java/org/jabref/gui/EntryTypeView.java

```diff
@@ -168,4 +178,119 @@ private void setEntryTypeForReturnAndClose(Optional<BibEntryType> entryType) {
         viewModel.stopFetching();
         this.close();
     }
+
+    //The description is coming from biblatex manual chapter 2
+    //Biblatex documentation is favored over the bibtex,
+    //since bibtex is a subset of biblatex and biblatex is better documented.
+    public String getDescription(BibEntryType selectedType) {
+        EntryType entryType = selectedType.getType();
+        try {
+            StandardEntryType entry = (StandardEntryType) entryType;
+            switch (entry) {
+                case Article -> {
+                    return Localization.lang("An article in a journal, magazine, newspaper, or other periodical which forms a self-contained unit with its own title.");
+                }
+                case Book -> {
+                    return Localization.lang("A single-volume book with one or more authors where the authors share credit for the work as a whole.");
+                }
+                case Booklet -> {
+                    return Localization.lang("A book-like work without a formal publisher or sponsoring institution.");
+                }
+                case Collection -> {
+                    return Localization.lang("A single-volume collection with multiple, self-contained contributions by distinct authors which have their own title. The work as a whole has no overall author but it will usually have an editor.");
+                }
+                case Conference -> {
+                    return Localization.lang("A legacy alias for inproceedings.");
```

Comment:
@dimitra-karadima Minor suggestion: e.g. `inproceedings` could be renamed to `InProceedings`. Reasoning: For consistency reasons, entry types which are documented with a preceding '@' (like @ proceedings) in the descriptional text for an entry type could be written in the same way, as they are selectable in the UI of JabRef (e.g. Conference: @ inproceedings -> InProceedings; MvBook: @ book -> Book; MasterThesis: @ thesis -> Thesis; ...)

Reply:
@dimitra-karadima For even better distiction between common text, you could also encapsulate those entry type phrases in ticks (e.g. `'InProceedings'`) or quotes (e.g. `"InProceedings"`), which makes it even more understandable. In the biblatex documentation this is done on the one hand with the `@` symbol and formatting it as some code, but encapsulating would be really nice. Then it is easy to distinguish e.g. the common text term theses from an the entry type 'Thesis' (which should reference to `@thesis`).

Reply:
mvBook stand for multi volume book

Reply:
@systemoperator So what do you suggest better? Ticks or quotes over the preceding @ ?

Reply:
@dimitra-karadima Please use quotes, as they are use for describing the entrie fields of an entry type as well (see: https://github.com/JabRef/jabref/pull/6239).